### PR TITLE
Use operating system notifications (#3778)

### PR DESCRIPTION
**Pull request by lhazenoot**
- In release note :
  -  In chapter :  Features
  -  Text : #3778

### CODE REVIEW

**Comment by lhazenoot** [comment]
@freddidierRTE 
To receive notifications, you need to manually allow notifications in the settings of your browser. There is no easy way to do this trough opfab.

**Comment by freddidierRTE** [comment]
it does not build

File: ui/main/src/app/business/services/system-notification.service.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        private lightCardsFeedFilterService: LightCardsFeedFilterService,
+        private lightCardsStoreService: LightCardsStoreService,
+        private configService: ConfigService,
+        private opfabEventStreamService: OpfabEventStreamService,
+        private alertMessageService: AlertMessageService,
+        private logger: OpfabLoggerService,
+        private router: Router
+    ) {
+        // use to have access from cypress to the current object for stubbing method sendNotification
+        if (window['Cypress']) window['systemNotificationService'] = this;
```
Comment (freddidierRTE) [inline ui/main/src/app/business/services/system-notification.service.ts:48]: We are not gonna test this service yet with cypress , so you can remove this line

File: ui/main/src/assets/i18n/en.json [hunk 2]
```diff
@@ -251,7 +251,12 @@
     "replayEnabled": "Sound replay enabled",
     "playSoundOnExternalDevice": "Play sound on external device",
     "replayInterval": "INTERVAL BETWEEN SOUND REPLAYS (SECONDS)",
-    "remoteLoggingEnabled": "Remote logging enabled"
+    "remoteLoggingEnabled": "Remote logging enabled",
+    "notificationOnOperatingSystem": "NOTIFICATIONS FOR CARDS WITH SEVERITY",
```
Comment (freddidierRTE) [inline ui/main/src/assets/i18n/en.json:255]: SYSTEM NOTIFICATION

File: ui/main/src/assets/i18n/fr.json [hunk 3]
```diff
@@ -251,7 +251,12 @@
     "replayEnabled": "Répétition des sons activée",
     "playSoundOnExternalDevice": "Jouer le son sur un appareil externe",
     "replayInterval": "DÉLAI ENTRE LES RÉPÉTITIONS (EN SECONDES)",
-    "remoteLoggingEnabled": "Journalisation à distance activée"
+    "remoteLoggingEnabled": "Journalisation à distance activée",
+    "notificationOnOperatingSystem": "NOTIFICATIONS POUR LES CARTES DE TYPE",
```
Comment (freddidierRTE) [inline ui/main/src/assets/i18n/fr.json:255]: NOTIFICATIONS SYSTEM POUR ...

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

**Pull request merged**